api/context/context.go

```diff
@@ -361,3 +365,29 @@ func (dF DefaultConfig) GetDB() db.Requests {
 	}
 	return &db.MongoRequests{}
 }
+
+// GetCache returns a new cache based on the HUSKYCI_CACHE_DEFAULT_EXPIRATION
+// and HUSKYCI_CACHE_CLEANUP_INTERVAL environment variables.
+func (dF DefaultConfig) GetCache() *cache.Cache {
+	var (
+		defaultExpiration time.Duration
+		cleanupInterval   time.Duration
+		err               error
+	)
+
+	defaultExpiration, err = time.ParseDuration(
+		dF.Caller.GetEnvironmentVariable("HUSKYCI_CACHE_DEFAULT_EXPIRATION"),
```

Comment:
Could you use `HUSKYCI_CACHE_DEFAULT_EXPIRATION_IN_MINUTES` instead, please? 😄

Reply:
But this values can be passed in the golang duration format (see [duration.ParseDuration](https://golang.org/pkg/time/#ParseDuration)).

Reply:
Ops, good point! Ok, we can explicitly use a few examples in documentation so that the users understand what type of parameters can be used here. Thanks, @lzakharov!